### DISCUSSION TITLE
Calibration Error in CWatM

### DISCUSSION THREAD

**Discussion by Matin64112** [iiasa/CWatM#165] [Q&A]
Servus,  Guten Tag 

My name is Matin and I am PhD student at the University of Oulu.
I am trynig to calibrate a **5-arcminute resolution CWatM model** on a **monthly time scale**. My "settings_calibration.txt": is as follows:

ForcingStart = 1/1/1932
ForcingEnd = 1/12/2007
timeperiod = daily
bestrun = True

and "settings_CWatM_template.ini :"

StepStart = 1/1/1926
SpinUp =  1/1/1932
StepEnd =  1/12/2007

OUT_Dir = %run_rand_id
OUT_TSS_Daily = discharge
OUT_TSS_MonthAvg = discharge

When running the calibration process, I encountered the following error:
![Screenshot 2024-11-27 121912](https://github.com/user-attachments/assets/55c90c11-95a5-4449-aa6f-6d8d4e60ecec)

and the log file says: 

![image](https://github.com/user-attachments/assets/4a09f795-855a-402b-a842-6168213dabe4)

I have also attached my settings files, and I changed the format of the .ini file to upload it. 
**I was wondering if  "settings_CWatM_template.ini"  is appropriate only for 30 arcmin, not 5 arcmin resolution? When I compare the settings file for 30 arcminutes, I find that the above-mentioned file is available.**

I would greatly appreciate any guidance or examples related to resolving this error. 
Thank you!  

[settings_calibration.txt](https://github.com/user-attachments/files/17932847/settings_calibration.txt)
[settings_CWatM_template.txt](https://github.com/user-attachments/files/17932909/settings_CWatM_template.txt)

**Comment by mikhailsmilovic** [comment]
Guten Tag Matin64112,

The settings file setups for 5 minutes and 30 minutes are slightly different. You are using the 30 min version.
There should be a template settings file in the folder with the 5 min input data. 
Let me know, and we go from there!
